seqr/views/apis/data_manager_api_tests.py

```diff
@@ -1054,15 +1053,18 @@ def _test_basic_data_loading(data, num_parsed_samples, num_loaded_samples, new_s
 
         # test correct file interactions
         file_path = RNA_FILENAME_TEMPLATE.format(data_type)
-        mock_subprocess.assert_has_calls([mock.call(command, stdout=-1, stderr=-2, shell=True) for command in [  # nosec
+        expected_subprocess_calls = [
             f'gsutil ls {RNA_FILE_ID}',
             f'gsutil cat {RNA_FILE_ID} | gunzip -c -q - ',
-            f'gsutil mv tmp/{file_path}/* gs://seqr-scratch-temp/{file_path}',
-        ]])
-        mock_mkdir.assert_called_with(f'tmp/{file_path}')
-        filename = f'tmp/{file_path}/{new_sample_guid}.json.gz'
+        ] + self._additional_expected_loading_subprocess_calls(file_path)
+        self.assertEqual(mock_subprocess.call_count, len(expected_subprocess_calls))
+        mock_subprocess.assert_has_calls([
+            mock.call(command, stdout=-1, stderr=-2, shell=True) for command in expected_subprocess_calls  # nosec
+        ])
+        mock_mkdir.assert_any_call(f'tmp/temp_uploads/{file_path}')
```

Comment:
is gettempdir() called multiple times in the code path under test? if applicable here, it could be better to use `assert_has_calls`

Reply:
so that is a good point, but under the hood there are some additional calls being made from inside `get_temp_file_path` to create the initial temp directory that are tested elsewhere and would be really confusing to test here, as they are called way more often than you would expect due to the directory not actually being created because of the mocking. Since that behavor is tested elsewhere and the only thing we really care about for this file is that the one call to `mkdir` in this file is called correctly, I thought it was neater just to test the relevant call